src/test/emulators/storage/files.spec.ts

```diff
@@ -173,3 +435,20 @@ describe("files", () => {
     const getPersistenceTmpDir = () => `${tmpdir()}/firebase/storage/blobs`;
   });
 });
+function getCurrentDate(): string {
+  return new Date().toISOString().replaceAll("-", "").replaceAll(":", "").replaceAll(".", "");
+}
+
+/**
+ * get the current date + changeBy SECONDS
+ * @date 7/12/2023 - 4:28:20 PM
+ *
+ * @param {number} changeBy
+ * @returns {string}
+ */
+function getAdjustedDate(changeBy: number): string {
+  const newDate = new Date();
+  const adjutedDate = new Date(newDate.getTime() + changeBy * SECONDS_TO_MS_FACTOR);
+
+  return adjutedDate.toISOString().replaceAll("-", "").replaceAll(":", "").replaceAll(".", "");
```

Comment:
You can add a helper that does:

```
function getSignedUrlTimestampFor(date: Date): string = {
   return date.toISOString().replaceAll("-", "").replaceAll(":", "").replaceAll(".", "")
}
```

And then call it everywhere you do this replacement (in tests + implementation).

Reply:
would this be doing the same thing but just passing in what the first 2 lines are calculating?

Reply:
I like the idea of the helper, but in my opinion `   return getSignedUrlTimestampFor(new Date(new Date().getTime() + changeBy * SECONDS_TO_MS_FACTOR));
` is hard to read and it isnt very obvious what is happening. I think I am confused on what should be inlined and what should be separated for readability purposes